thrift/history.thrift

```diff
@@ -456,6 +458,7 @@ service HistoryService {
       6: shared.DomainNotActiveError domainNotActiveError,
       7: shared.LimitExceededError limitExceededError,
       8: shared.ServiceBusyError serviceBusyError,
+      9: shared.WorkflowExecutionAlreadyCompletedError workflowExecutionAlreadyCompletedError,
```

Comment:
This and `RecordDecisionTaskStarted` will also affect the error handling logic in matching.

Reply:
Could you give more details? Could you point me to the places in matching if we need to address.

Reply:
never mind, found them, will update the other PR.